.github/labeler.yml

```
@@ -0,0 +1,53 @@
+﻿site:content:
+  - changed-files:
+      - any-glob-to-any-file: content/wiki/**
```

nooo

```suggestion
      - any-glob-to-any-file: content/**
```

i plan on adding other branches such as `site` which would host the ToS and Privacy Policy. and maybe a `blog` and move from posting galaxypedia updates on my personal blog to the galaxypedia blog